code/modules/mob/living/silicon/pai/pai.dm

```diff
@@ -36,6 +36,10 @@
 		"Parrot" = "pai-parrot",
 		"Rabbit" = "pai-rabbit",
 		//VOREStation Addition Start
+		"Dire wolf" = "pai-diredog"
+		"Horse (Lune)" = "pai-horse_lune"

```

Comment:
Ah, you need to ass commas at the end of list entries like the others

Reply:
Ah good catch, I had the feeling I was missing something really obvious like that.